tests/virt/node/node_labeller/cpu_features/test_vm_with_cpu_features.py

```diff
@@ -5,20 +5,9 @@
 import pytest
 from kubernetes.dynamic.exceptions import UnprocessibleEntityError
 
-from utilities.constants import AMD
 from utilities.virt import VirtualMachineForTests, fedora_vm_body, running_vm
 
-pytestmark = [
-    pytest.mark.post_upgrade,
-    pytest.mark.sno,
-    pytest.mark.usefixtures("skip_if_amd_cpu_nodes"),
-]
-
-
-@pytest.fixture()
-def skip_if_amd_cpu_nodes(nodes_cpu_vendor):
-    if nodes_cpu_vendor == AMD:
-        pytest.skip("PCID CPU feature is not supported on AMD CPU")
+pytestmark = [pytest.mark.post_upgrade, pytest.mark.sno]
```

Comment:
and here - I would fail the test run if it is an AMD cluster

Reply:
added back